core/src/View/Kokkos_ViewTracker.hpp

```diff
@@ -62,7 +62,7 @@ struct ViewTracker {
   }
 
   template <class RT, class... RP>
-  KOKKOS_INLINE_FUNCTION void assign(const View<RT, RP...>& vt) noexcept {
+  KOKKOS_INLINE_FUNCTION void assign(const View<RT, RP...>& vt) {
```

Comment:
having to remove the `noexcept` seems unrelated ... would it make sense to put it into a different pr and merge that one first?

Reply:
I can split it further but the pull request is already very small in terms of LOC.

Reply:
What is the reasoning between removing `noexcept`?

Reply:
`clang-tidy` complains that thsi function can throw.

Reply:
I'd vote keeping in one PR, seems related enough to the testing updates in the PR.